tests/Mindee.IntegrationTests/MindeeClientTest.cs

```diff
@@ -44,12 +50,10 @@ public async Task Parse_File_SinglePage_MustSucceed()
         }
 
         [Fact]
-        public async Task Parse_Url_SinglePage_MustSucceed()
+        public async Task Parse_Url_Standard_SinglePage_MustSucceed()
         {
-            var apiKey = Environment.GetEnvironmentVariable("Mindee__ApiKey");
-            var mindeeClient = new MindeeClient(apiKey);
             var inputSource = new UrlInputSource("https://raw.githubusercontent.com/mindee/client-lib-test-data/main/products/expense_receipts/default_sample.jpg");
```

Comment:
Never noticed this one. I think I'll steal it :smiling_imp:

Reply:
:+1: